lib/gitarro/backend.rb

```diff
@@ -19,13 +19,11 @@ def initialize(options)
 
```

Comment:
Most changes at this file are refactors so the class can have less lines and rubocop does not complain.

At some point we should refactor the whole class to separate it into different classes.